package.json

```diff
@@ -5,15 +5,16 @@
   "main": "dist/main.js",
   "scripts": {
     "clean": "rm -rf dist",
-    "start": "ulimit -n 999999 || ulimit -n 2048; npm run build && node dist/main.js -p 8080 --log-level=debug",
```

Comment:
Why did the `npm run build` get removed?

Reply:
The build is done during deploy by the `npm install` and `postinstall` step. We cannot build on the GAE server as the file system is read-only.

Reply:
Cool!